client/app/states/requests/list/list.state.js

```diff
@@ -43,7 +43,7 @@
   }
 
   /** @ngInject */
-  function StateController($state, requests, orders, RequestsState, $filter, $rootScope, lodash) {
+  function StateController($state, requests, orders, RequestsState, OrdersState, $filter, $rootScope, lodash) {
```

Comment:
This is a generic comment ... not for this PR per se.
What I've noticed is, the inject list keeps on growing...I think the linter will warn after a cap of 10 is reached. 
Do you think at that point we should consolidate the parameters and then pass them as arrays?
For e.g. Consolidate all states in a `States` Array